.spin/cmds.py

```diff
@@ -9,9 +9,21 @@
 
 @click.command()
 @click.option(
-    "--clean", is_flag=True, help="Clean previously built docs before building"
+    "--clean", is_flag=True,
+    default=False,
+    help="Clean previously built docs before building"
 )
-def docs(clean=False):
+@click.option(
+    "--raise-on-warning/--no-raise-on-warning",
```

Comment:
Might be worth turning this option into `--sphinx-opts` that accepts a string with the default being `'-W'`?